src/app/exhibitsPage/page.tsx

```diff
@@ -0,0 +1,80 @@
+'use client';
+
+import React, { useEffect, useState } from 'react';
+import Link from 'next/link';
+import NavBar from '../../components/userComponents/navBar/navBar';
+import { CategoryRow } from '../../types/types';
+import { fetchAllCategories } from '../../supabase/category/queries';
+import Exhibit from '../../components/userComponents/Exhibit/Exhibit';
+import BackButton from '../../components/userComponents/BackButton/page';
+
+/**
```

Comment:
More comprehensive comments for this page and what's happening in useEffect.  Assume minimal background knowledge of this sprint for handoff